### PR TITLE
fix: .expect_result must try, regression from 2.4.0.1.0

### PR DESCRIPTION
**Pull request by kantai**
### Description

This fixes (and adds a regression test) for a regression in 2.5.0.0.0 branch from 2.4.0.1.0

### CODE REVIEW

**Review by obycode — APPROVED** [review]
lgtm

**Review by jcnelson — APPROVED** [review]

File: clarity/src/vm/contexts.rs [hunk 1]
```diff
@@ -2119,6 +2125,35 @@ mod test {
         assert_eq!(table[&p2][&t7], AssetMapEntry::Burn(35 + 36));
     }
 
+    /// Test the stx-transfer consolidation tx invalidation
+    ///  bug from 2.4.0.1.0
```
[the remark this answers is not in the document]
Comment (wileyj) [reply clarity/src/vm/contexts.rs]: updated in https://github.com/stacks-network/stacks-core/commit/63bafb0180b7d28be71c0651a96349618e6d42a3

**Review by obycode — APPROVED** [review]

**Review by wileyj — APPROVED** [review]
shipit

File: clarity/src/vm/contexts.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (wileyj) [reply clarity/src/vm/contexts.rs]: @jcnelson resolving this so we're not blocked. please ping if you still have a nit here.

**Pull request merged**